core/db/migrate/20150515211137_fix_adjustment_order_id.rb

```diff
@@ -0,0 +1,75 @@
+class FixAdjustmentOrderId < ActiveRecord::Migration
+  def change
+    say 'Populate order_id from adjustable_id where appropriate'
+    execute(<<-SQL.squish)
+      UPDATE
+        spree_adjustments
+      SET
+        order_id = adjustable_id
+      WHERE
+            adjustable_type = 'Spree::Order'
+        AND order_id IS NULL
+      ;
+
+      UPDATE
+        spree_adjustments
+      SET
+        order_id =
+          (SELECT order_id FROM spree_line_items WHERE spree_line_items.id = spree_adjustments.adjustable_id)
+      WHERE
+            adjustable_type = 'Spree::LineItem'
+        AND order_id IS NULL
+      ;
+
+      UPDATE
+        spree_adjustments
+      SET
+        order_id =
+          (SELECT order_id FROM spree_shipments WHERE spree_shipments.id = spree_adjustments.adjustable_id)
+      WHERE
+            adjustable_type = 'Spree::Shipment'
+        AND order_id IS NULL
+    SQL
```

Comment:
> This is failing for me under mysql

Can you show me how it failed?

> (which our CircleCI unfortunately doesn't test)

Lets fix it. We already use multiple containers, I could easily change the `build-ci.rb` script to run each projects also against a different `DB` environment variable.

This would double the CI times, or in case you double containers would be at free (time) cost.

I know this is not ideal, but an improvement.